payments-core/src/androidTest/java/com/stripe/android/TestActivity.kt

```diff
@@ -0,0 +1,9 @@
+package com.stripe.android
+
+import androidx.annotation.RestrictTo
+import androidx.annotation.VisibleForTesting
+import androidx.fragment.app.FragmentActivity
+
+@VisibleForTesting
```

Comment:
These annotations aren't needed now since it's in the test directory already (and internal).

Reply:
Thanks! Updated